app/frontend/src/components/designer/FormViewer.vue

```diff
@@ -706,14 +706,16 @@ export default {
             case 'custom':
               if (obj.component.validate.custom.length) {
                 validatorIdentity +=
-                  '|custom:' + obj.component.validate[validity].trim();
+                  '|custom:' +
+                  obj.component.validate[validity].trim().replaceAll(',', ';');
```

Comment:
No it should not break any form but I was breaking the reports sheet (.csv) generated for the validation report. So We replaced , with ; so that CSV report file don't get messed